Runtime/src/index.ts

```diff
@@ -127,28 +121,28 @@ export class SwiftRuntime {
             payload1: number,
             payload2: number
         ) => {
-            const obj = this.memory.getObject(ref);
-            const key = this.memory.getObject(name);
-            const value = JSValue.decode(kind, payload1, payload2, this.memory);
+            const memory = this.memory;
+            const obj = memory.getObject(ref);
+            const key = memory.getObject(name);
+            const value = JSValue.decode(kind, payload1, payload2, memory);
             obj[key] = value;
         },
         swjs_get_prop: (
             ref: ref,
             name: ref,
-            kind_ptr: pointer,
             payload1_ptr: pointer,
             payload2_ptr: pointer
         ) => {
-            const obj = this.memory.getObject(ref);
-            const key = this.memory.getObject(name);
+            const memory = this.memory;
+            const obj = memory.getObject(ref);
+            const key = memory.getObject(name);
             const result = obj[key];
-            JSValue.write(
+            return JSValue.writeV2(
```

Comment:
Is the old version of `write` still used anywhere? Should it be renamed to `writeV1` then?

Reply:
The old one is still used for writing out multiple values (e.g. passing js values to Swift closure )